prompts.py

```diff
@@ -1,69 +1,117 @@
 import trading_algorithm
+import os
+import readline
+from pick import pick
 
-def prompt_user():
-  '''Prompts User for input to start algorithm, returns trading_terms. 
-  '''
-  ready = True
-  print(" ***********************Trading Algorithm************************ \
-    \n ***********************By William P. Fey************************ \
-    \n\n" +\
-    "This program will list a squence of buy and sell trades on \n"+ \
-    "GDAX.com based on the follosing input.\n")
-  while ready:
+def _input_default(prompt, default):
+  def hook():
+    readline.insert_text(default)
+    readline.redisplay()
+  readline.set_pre_input_hook(hook)
+  result = input(prompt)
+  readline.set_pre_input_hook()
+  return result
+
+def _prompt_float(title, default):
```

Comment:
generic function to prompt user for a float value.  will yell at them if supplied value is not a float